src/Article.jsx

```diff
@@ -0,0 +1,94 @@
+import { useState, useEffect } from "react";
+import { getArticle, patchArticleVote } from '../utils/api';
+import * as React from 'react';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import { useParams } from "react-router-dom";
+import CommentsList from "./CommentsList";
+import Stack from '@mui/material/Stack';
+import Button from '@mui/material/Button';
+
+
+const Article = () => {
+    const [article, setArticle] = useState({});
+    const [isLoading, setIsLoading] = useState(true);
+    const [isError, setIsError] = useState(false)
+    const { article_id } = useParams();
+    const [showComments, setShowComments] = useState(false);
+    const [addVote, setAddVote] = useState(0);
+
+    const handleLikeClick = () => {
+        article.votes++
```

Comment:
Your mutating state here, so will possibly have react issues. 

https://medium.com/@kkranthi438/dont-mutate-state-in-react-6b25d5e06f42